src/styles/buy.scss

```diff
@@ -0,0 +1,28 @@
+.buy {
+  display: flex;
+  box-sizing: border-box;
+  width: $defaultWidth;
+  height: $buttonHeight;
+  background-color: $buttonColor;
```

Comment:
The variable `$buttonColor` is used but not defined. Ensure it is defined in your SCSS files to avoid compilation errors.